pallets/creditcoin/src/tests.rs

```diff
@@ -3236,3 +3237,47 @@ fn register_address_v2_should_error_with_unsupported_blockchain() {
 		);
 	});
 }
+
+use hex_literal::hex;
+
+#[test]
+fn set_burn_gate_contract_should_return_default_goerli_contract_when_not_set() {
```

Comment:
```suggestion
fn gate_contract_storage_should_return_default_goerli_contract_when_not_set() {
```

because this test has nothing to do with the `set_gate_contract()` extrinsic.